homeassistant/components/light/http_rgb.py

```diff
@@ -0,0 +1,106 @@
+"""
+Support for http_rgb lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.http_rgb/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, ATTR_RGB_COLOR, SUPPORT_RGB_COLOR,
+    Light, PLATFORM_SCHEMA)
+from homeassistant.const import (CONF_HOST, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['http_rgb==0.1.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'http_rgb'
+
+SUPPORT_HTTP_RGB = (SUPPORT_BRIGHTNESS | SUPPORT_RGB_COLOR)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the http_rgb Light platform."""
+    import http_rgb
+
+    name = config.get(CONF_NAME)
+    host = config.get(CONF_HOST)
+
+    light = http_rgb.http_rgb(host)
```

Comment:
I assume that there will be an exception if the device is not available. You need to decide what to do here. Using `PlatformNotReady` or make the setup fail.